### DISCUSSION TITLE
Integration with Griptape

### DISCUSSION THREAD

**Discussion by Ioannis-Pikoulis** [NVIDIA-NeMo/Guardrails#146] [Q&A]
Hello.

Is there any plan integrating `NeMo-Guardrails` with [`Griptape`](https://github.com/griptape-ai/griptape) (Enterprise counterpart of `LangChain`)?

**Comment by drazvan** [comment]
HI @Ioannis-Pikoulis. Not at the moment, but thanks for pointing this. I've added to our internal list to prioritise.
